README.md

```diff
@@ -274,6 +274,7 @@ Additional lists you might find useful:
 ## REST and API
 *Plugins and web tools for developing REST-ful APIs.*
 
+- [API Pagination Plugin](https://github.com/bcrowe/cakephp-api-pagination) - Plugin that injects pagination information into serialized JsonView and XmlView responses.
```

Comment:
The plugin name looks wrong. And you can directly start with `Injects ...` as its clear then it is a plugin.